sphinx_parser/output.py

```diff
@@ -49,67 +47,56 @@ def en_split(e, counter=energies[:, 0]):
     return results
 
 
-def collect_residue_dat(file_name="residue.dat", cwd="."):
+def collect_residue_dat(file_name="residue.dat"):
     """
 
     Args:
         file_name (str): file name
-        cwd (str): directory path
 
     Returns:
         (dict): results
 
     """
-    if cwd is None:
-        cwd = "."
-    residue = np.loadtxt(str(Path(cwd) / Path(file_name)), ndmin=2)
+    residue = np.loadtxt(file_name, ndmin=2)
     if len(residue) == 0:
         return {}
     return {"scf_residue": _splitter(residue[:, 1:].squeeze(), residue[:, 0])}
 
 
-def _collect_eps_dat(file_name="eps.dat", cwd=None):
+def _collect_eps_dat(file_name="eps.dat"):
     """
 
     Args:
         file_name:
-        cwd:
 
     Returns:
 
     """
-    path = Path(file_name)
-    if cwd is not None:
-        path = Path(cwd) / path
-    return np.loadtxt(str(path), ndmin=2)[..., 1:]
+    return np.loadtxt(str(file_name), ndmin=2)[..., 1:]
 
 
-def collect_eps_dat(file_name=None, cwd=None, spins=True):
+def collect_eps_dat(file_name=None, spins=True):
     if file_name is not None:
-        values = [_collect_eps_dat(file_name=file_name, cwd=cwd)]
+        values = [_collect_eps_dat(file_name=file_name)]
     elif spins:
-        values = [_collect_eps_dat(file_name=f"eps.{i}.dat", cwd=cwd) for i in [0, 1]]
+        values = [_collect_eps_dat(file_name=f"eps.{i}.dat") for i in [0, 1]]
     else:
-        values = [_collect_eps_dat(file_name="eps.dat", cwd=cwd)]
+        values = [_collect_eps_dat(file_name="eps.dat")]
```

Comment:
@samwaseda This only works when the Python code is executed in the output directory of the sphinx job, right?